Changelog.md

```diff
@@ -1,3 +1,10 @@
+## Unreleased
+
+### Added
+
+-   Support for apps to open a new app, and automatically connecting to a device
+    and a serial port.
```

Comment:
```suggestion
-   Support for apps to open a new app, and automatically connect to a device
    and a serial port.
```

Reply:
I suggest to not mention it at all, as it seems more an internal infrastructure. Users do not really care how the launcher enables apps to do their thing.

Reply:
Removed the entry